bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/things/VeluxKLFAPI.java

```diff
@@ -129,7 +129,7 @@ public String toString() {
         UNDEFTYPE((short) -1, "Unknown command."),
         // Special item: Shutdown of the connection
         GW_OPENHAB_CLOSE((short) -2, "openHAB connection shutdown command."),
-        // Special item: Shutdown of the connection
+        // Special item: ???
```

Comment:
Empty Command (used to send nothing and process an incoming message only).

Reply:
Thank you. Suggestion adopted.